src/components/Link/Link.jsx

```diff
@@ -50,24 +52,27 @@ const Link = forwardRef(
         ref={ref}
         onClick={onClickWrapper}
         target={target}
-        className={cx("monday-style-link", overrideClassName, {
-          "inherit-font-size": inheritFontSize,
-          "inline-text": inlineText
+        className={cx(styles.link, "monday-style-link", overrideClassName, {
+          [styles.inheritFontSize]: inheritFontSize,
+          ["inherit-font-size"]: inheritFontSize,
+          [styles.inlineText]: inlineText,
+          ["inline-text"]: inlineText
         })}
         aria-label={ariaLabelDescription}
         aria-labelledby={ariaLabeledBy}
+        data-testid={dataTestId || getTestId(ELEMENT_TYPES.LINK, id)}
       >
-        {getIcon(isStart, icon, "monday-style-link--icon-start")}
-        <span className="monday-style-link--text">{text}</span>
-        {getIcon(!isStart, icon, "monday-style-link--icon-end")}
+        {getIcon(isStart, icon, cx(styles.iconStart, "monday-style-link--icon-start"))}
+        <span className={cx(styles.linkText, "monday-style-link--text")}>{text}</span>
+        {getIcon(!isStart, icon, cx(styles.iconEnd, "monday-style-link--icon-end"))}
       </a>
     );
   }
 );
 
 function getIcon(shouldShow, icon, className) {
   if (!shouldShow) return;
-  return <Icon className={className} clickable={false} icon={icon} iconType={Icon.type.ICON_FONT} />;
+  return <Icon className={cx(className)} clickable={false} icon={icon} iconType={Icon.type.ICON_FONT} />;
```

Comment:
maybe we can remove the cx wrapper when only one class is passed to the class name?